lib/rubocop/rails/schema_loader.rb

```diff
@@ -7,27 +7,29 @@ module Rails
     module SchemaLoader
       extend self
 
-      # It parses `db/schema.rb` and return it.
-      # It returns `nil` if it can't find `db/schema.rb`.
+      # It parses schema file at  and return it.
+      # It returns `nil` if it can't find the schema file.
       # So a cop that uses the loader should handle `nil` properly.
+      # @param target_ruby_version [String] The target Ruby version
+      # @param schema_path [String] The path to the schema file, defaults to `db/schema.rb`
       #
       # @return [Schema, nil]
-      def load(target_ruby_version)
+      def load(target_ruby_version, schema_path = 'db/schema.rb')
         return @load if defined?(@load)
 
+        @schema_path = schema_path
```

Comment:
Question: should I guard against setting `@schema_path` to `nil` here? Should I always set a value?

e.g.

```suggestion
@schema_path = schema_path || 'db/schema.rb'
```